app/views/users/show.html.erb

```diff
@@ -20,7 +20,7 @@
       color: #cedae2;
     }
     .user-profile-header .user-profile-header-container .profile-details h1{
-      color: <%= HexComparer.new([user_colors(@user)[:bg], user_colors(@user)[:text]]).brightness(0.78) %> !important;
```

Comment:
Same here, I think we should just drop this style entirely as well as customization below for night theme: https://github.com/thepracticaldev/dev.to/pull/8406/files#diff-c7c9a522f39f5d8cd9b512cd928b2d14L25-L27